earth_enterprise/src/fusion/gecombineterrain/gecombineterrain.cpp

```diff
@@ -88,7 +88,9 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
       "      --help | -?:  Display this usage message\n"
       "      --sortbuf <buffer_size>: size of packet index sort buf in MB\n"
       "        (default %u)\n"
-      "      --numcpus:    Number of CPUs to use (default %u)\n"
+      "      --numCompressThreads:    Number of CPUs/threads to use \n("
```

Comment:
Minor: The command-line option sets the number of threads that will be used.  We don't know how many CPUs the underlying schedulers will use.